modules/network-observability-flowmetrics-api-specifications.adoc

```diff
@@ -0,0 +1,298 @@
+// Automatically generated by 'openshift-apidocs-gen'. Do not edit.
+:_mod-docs-content-type: REFERENCE
+[id="flowmetric-flows-netobserv-io-v1alpha1"]
+= FlowMetric [flows.netobserv.io/v1alpha1]
```

Comment:
@jotak I just want to make sure, this is FlowMetric API, not FlowMetrics API? If there's no "s", I need to update the feature doc: https://74412--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/network_observability/metrics-alerts-dashboards.html#network-observability-custom-metrics_metrics-dashboards-alerts

Reply:
Technically speaking both are corrects, the API objects can be referred to with or without plural.
But I would stand without "s" just because this is what our docgen tool produces

Reply:
Okay, I will open a PR to make this consistent

Reply:
PR: https://github.com/openshift/openshift-docs/pull/76902